src/main/kotlin/day05/Encapsulation.kt

```
@@ -0,0 +1,27 @@
+package org.example.day05
+
+class Person(
+    var name: String,
+    var age: Int,
+) {
+    fun bye() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)